### ISSUE TITLE
Vote command clickable links are broken

### ISSUE THREAD

**Issue by uncovery** [uncovery/uncovery_me#307]

**Comment by LukeBillington** [comment]
Also heads up, mineservers.net an minestatus.net are now minecraftservers.biz, these are also listed on the website.

**Comment by uncovery** [comment]
thanks, that's a separate issue.

**Comment by LukeBillington** [comment]
Hey there, sorry to reopen, minecraftservers.net has also not appeared to be working either for quite some time (the human verification doesn't work) to be able to vote. Suggest commenting it out.

**Comment by stayen** [comment]
I agree for minecraftservers.net - it asks arithmetic CAPTCHA, but then complains that the image one wasn't solved.

**Comment by uncovery** [comment]
ok, removed the vote and fixed the money, please test

**Comment by uncovery** [comment]
tested, seems to work.
